database/youtube/competitive-programming.json

```
@@ -0,0 +1,26 @@
+[{
+    "name": "Errichto",
+    "description": "This channel provides competitive programming tactics and thought process for advanced alogrithms .",
+    "url": "https://www.youtube.com/@Errichto",
+    "category": "youtube",
+    "subcategory": "competitive-programming",
+    "language": "english"
+},
+{
+    "name": "Priyansh Agarwal",
+    "description": "This channel provides free video solutions of coding contests of codeforces, codechef etc.",
```

[Proofreading] This description is a bit unclear. What do you mean by “ free video solutions of coding contests of codeforces, codechef, etc“?
```suggestion
    "description": "This channel provides free video solutions of coding contests of codeforces, codechef, etc."
```